care/utils/evaluators/evaluation_metric/base.py

```
@@ -13,30 +13,30 @@ def validate_rule(cls, operation, value):
+    def apply_rule(self, operation, rule, facility=None):
         if not self.context_object:
             return False
         if operation not in self.allowed_operations or not getattr(
             self, f"evaluate_{operation}", None
         ):
             raise ValueError("Invalid operation")
-        return getattr(self, f"evaluate_{operation}")(rule)
+        return getattr(self, f"evaluate_{operation}")(rule, facility)
 
-    def evaluate_equality(self, rule):
+    def evaluate_equality(self, rule, facility=None):
```

Ruff (0.14.3)

Unused method argument: `facility`

(ARG002)